mdal/frmts/mdal_cf.hpp

```diff
@@ -60,8 +60,9 @@ namespace MDAL
     };
     std::string name; //!< Dataset group name
     CFDimensions::Type outputType;
-    bool is_vector;
-    bool is_polar;
+    bool isVector;
+    bool isPolar;
+    bool isInvertedDirection;
```

Comment:
maybe rename to `hasInvertedPolarDirection`? (i assume it is applicable only for polar dir?)